GW2EIBuilders/HtmlModels/LogDataDto.cs

```diff
@@ -130,11 +134,52 @@ private LogDataDto(ParsedEvtcLog log, bool light, Version parserVersion, string[
         }
 
         EncounterDuration = log.FightData.DurationString;
+        EvtcRecordingDuration = log.FightData.EvtcRecordingDuration;
         Success = log.FightData.Success;
         Wvw = log.FightData.Logic.ParseMode == FightLogic.ParseModeEnum.WvW;
         Targetless = log.FightData.Logic.Targetless;
+        FightNameNoMode = log.FightData.FightNameNoMode;
         FightName = log.FightData.FightName;
         FightIcon = log.FightData.Logic.Icon;
+        switch (log.FightData.FightMode)
+        {
+            case FightData.EncounterMode.Story:
+                FightMode = "Story Mode";
+                break;
+            case FightData.EncounterMode.StoryCM:
+                FightMode = "Story Challenge Mode";
+                break;
+            case FightData.EncounterMode.NormalEmboldened:
+                FightMode = "Emboldened Normal Mode";
+                break;
+            case FightData.EncounterMode.Normal:
+                FightMode = "Normal Mode";
```

Comment:
To continue comment regarding emboldened, check InstanceBuffs here and put correct string accordingly.

Reply:
You can even add the number of stacks if you wish to do so.